lib/xboxrt/libc_extensions/direct.h

```diff
@@ -3,3 +3,13 @@
 // SPDX-FileCopyrightText: 2020 Jannik Vogel
 
 // Part of Microsoft CRT
+
+int _chdir(char* path);
+int _chdrive(int drive);
+char* _getcwd(char* buffer, size_t length);
+char* _getwd(char* path_name);
+int _getdrive(void);
+int _mkdir(const char* pathname);
+int _rmdir(const char* pathname);
+void _fnmerge(char* path, const char* drive, const char* dir, const char* name, const char* ext);
+char* _searchpath(const char* file);
```

Comment:
NIT: Missing newline at end of file, otherwise git isn't praising you.

Reply:
Should be resolved by 0b37066